bndtools.core/src/bndtools/wizards/project/ImportBndWorkspaceWizard.java

```diff
@@ -0,0 +1,451 @@
+                        this.workbench.showPerspective("bndtools.perspective", window);
+                    }
+                }
+            } catch (WorkbenchException e) {
+                error("Unable to switch to BndTools perspective", e);
+            }
+            result = true;
+        }
+        return result;
+    }
+
+    private void deleteOldProjectFiles(final Path projectPath) throws IOException {
+        final Path settings = projectPath.resolve(".settings");
+        if (Files.exists(settings)) {
+            Files.walkFileTree(settings, new SimpleFileVisitor<Path>() {
+                @Override
+                public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
+                    Files.delete(file);
+                    return FileVisitResult.CONTINUE;
+                }
+
+                @Override
+                public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOException {
+                    Files.delete(dir);
+                    return FileVisitResult.CONTINUE;
+                }
+            });
+        }
+
+        final Path project = projectPath.resolve(".project");
+        Files.deleteIfExists(project);
+        final Path classpath = projectPath.resolve(".classpath");
+        Files.deleteIfExists(classpath);
+    }
+
+    private boolean importProjects(File rootImportPath, IProgressMonitor monitor) throws Exception {
+        Workspace bndWorkspace = Workspace.getWorkspace(rootImportPath);
+
+        int steps = bndWorkspace.getAllProjects().size() + 2;
+
+        monitor.beginTask("Importing Bnd workspace", steps);
+
+        importConfigurationProject(rootImportPath, monitor);
+
+        // Import Projects
+        for (Project bndProject : bndWorkspace.getAllProjects()) {
+            importBndProject(bndProject, bndWorkspace, monitor);
+        }
+
+        // build
+        monitor.subTask("Building workspace");
+        ResourcesPlugin.getWorkspace().build(IncrementalProjectBuilder.CLEAN_BUILD, monitor);
+        monitor.worked(1);
+        monitor.done();
+
+        return true;
+    }
+
+    private void importConfigurationProject(File rootImportPath, IProgressMonitor monitor) throws Exception {
+        monitor.subTask("Import configuration project 'cnf'.");
+        final IWorkspace eclipseWorkspace = ResourcesPlugin.getWorkspace();
+        final IWorkspaceRoot eclipseWorkspaceRoot = eclipseWorkspace.getRoot();
+        final Workspace bndWorkspace = Workspace.getWorkspace(rootImportPath);
+
+        // Prepare Eclipse Workspace
+        updateEclipseWorkspaceSettings(bndWorkspace);
+
+        // create generic project
+        final IProjectDescription cnfProjectDescription = eclipseWorkspace.newProjectDescription(Workspace.CNFDIR);
+        final IProject project = eclipseWorkspaceRoot.getProject(Workspace.CNFDIR);
+
+        deleteOldProjectFiles(Paths.get(bndWorkspace.getBase().toURI()).resolve(Workspace.CNFDIR));
```

Comment:
There were some problems with the project-settings when old one are not deleted. For example the classpath was messed up.
Just give it a try with this method commented out and have a look at the generated projects (build path). 

If we can solve this, I am with you that the settings should be kept. I could add a checkbox "clear old settings" because sometimes it nice to start fresh.